.github/workflows/release-please.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
         max-parallel: 1
         matrix: 
-          actions: ["setupGcloudSDK", "appengine-deploy","deploy-cloudrun", "setup-gcloud", "get-secretmanager-secrets", "upload-cloud-storage"]
+          actions: ["setupGcloudSDK", "appengine-deploy","deploy-cloudrun", "setup-gcloud", "get-secretmanager-secrets", "upload-cloud-storage", "deploy-cloud-functions"]
```

Comment:
Does line 35 and 51 need this as well or are you skipping those actions for now?

Reply:
Skipping releases for now due to repo move. We can cut each actions new release after the move